### PR TITLE
Preview images on hover

### PR DESCRIPTION
**Pull request by uiureo**
[Gyazo](https://gyazo.com/77042063b28ac40c43130ee73fc9cf64)

### CODE REVIEW

**Comment by uiureo** [comment]
@pastak @rakusai Can you review?

File: src/common/content/expander/injectCss.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    transform: rotate(360deg);
+  }
+}
+`
+
+module.exports = function () {
+  const node = document.createElement('style')
+  node.innerHTML = css
+  document.body.appendChild(node)
+}
```
Comment (pastak) [inline src/common/content/expander/injectCss.js]: Why don't you use `chrome.tabs.insertCSS`?

Comment (uiureo) [inline src/common/content/expander/injectCss.js]: because `chrome.tabs` api is not available in this context

Comment (uiureo) [inline src/common/content/expander/injectCss.js]: Should I write injectCSS in here?
https://github.com/gyazo/gyazo-chrome-extension/blob/master/src/normal/index.js#L132

Comment (pastak) [inline src/common/content/expander/injectCss.js]: It's correct position :smile:

Comment (uiureo) [inline src/common/content/expander/injectCss.js]: ok

File: src/common/content/expander/lib/adjacentStyle.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      left: rect.left,
+      bottom: Math.round((window.innerHeight - rect.top + offsetY)),
+      maxHeight: Math.round(Math.min(rect.top - offsetY * 2, 500))
+    }
+  } else {
+    const rectBottom = rect.top + rect.height
+    return {
+      left: rect.left,
+      top: Math.round(rectBottom + offsetY),
+      maxHeight: Math.round(Math.min(window.innerHeight - rectBottom, 500))
```
Comment (rakusai) [inline src/common/content/expander/lib/adjacentStyle.js]: should add ` - offsetY * 2` into `window.innerHeight - rectBottom - offsetY * 2`

Comment (uiureo) [inline src/common/content/expander/lib/adjacentStyle.js]: good catch

**Comment by rakusai** [comment]
Except for the css it looks nice!

**Comment by pastak** [comment]
LGTM!

**Comment by uiureo** [comment]
Thanks

**Pull request merged**